dependencies-py3.dt.txt

```diff
@@ -3,3 +3,5 @@
 
 # Duckietown World
 duckietown-world-daffy==6.4.3
+
+lib-dt-computer-vision==0.1.7
```

Comment:
@afdaniele since we install this here we can remove `libraries/lib-dt-computer-vision`, right?